doc/wiki.txt

```diff
@@ -238,13 +238,31 @@ CONFIGURATION                                                     *wiki-config*
         \ },
         \}
 
-*g:wiki_mappings_use_defaults*
+
+*g:wiki_mappings_use_all_defaults*
   Whether or not to use default mappings (see |wiki-mappings-default|). If
+  Whether or not to use all default mappings (see |wiki-mappings-default|). If
+  this is set to 0, then only custom mappings will be applied. Custom mappings
   this is set to 0, then only custom mappings will be applied. Custom mappings
   may either be specified explicitly or through |g:wiki_mappings_global| and
+  may either be specified explicitly or through |g:wiki_mappings_global| and
   |g:wiki_mappings_local|.
+  |g:wiki_mappings_local|.
+
 
   Default: 1
+  Default: 1
+
+
+*g:wiki_mappings_use_global_defaults*
+*g:wiki_mappings_use_local_defaults*
```

Comment:
Perhaps we should collect the docs for all three variants? Also, perhaps it would be better to call it `g:wiki_mappings_use_defaults_{all,local,global}`?

Reply:
Or, better: Instead of three options, how about a single option with different allowed values? It seems it could be enough with the values `'none', 'all', 'local', 'global'`, right?

Reply:
Seems like a good plan, yep.

Reply:
Fixed in a06e342